### PR TITLE
Add a tutorial showing how to set up an email alert in Prefect Cloud

### PR DESCRIPTION
**Pull request by daniel-prefect**
Closes https://linear.app/prefect/issue/DOC-102/create-an-operator-tutorial-for-managing-an-alert

Preview: https://prefect-bd373955-add_an_alerts_tutorial.mintlify.app/v3/tutorials/alerts

This PR also clarifies that when setting a workspace with the Prefect CLI, you need the account handle. I've updated the tutorials which use this command.

### Checklist

- [X] This pull request references any related issue by including "closes `<link to issue>`"
- [ ] ~If this pull request adds new functionality, it includes unit tests that cover the changes~
- [ ] ~If this pull request removes docs files, it includes redirect settings in `mint.json`.~
- [ ] ~If this pull request adds functions or classes, it includes helpful docstrings.~

### CODE REVIEW

File: docs/v3/tutorials/alerts.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+title: Send alerts to Slack on failure
```
Comment (EmilRex) [inline docs/v3/tutorials/alerts.mdx:2]: Slack alerts are certainly very common, but starting with an email alert might be a better choice as it requires no 3rd party setup. Plus you can send emails directly to a Slack channel to circumvent generating a token.

Comment (EmilRex) [inline docs/v3/tutorials/alerts.mdx:2]: Here's an example of defining an automation in Python code: https://github.com/PrefectHQ/examples/blob/main/scripts/create-automation.py

File: docs/v3/tutorials/alerts.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+1. Sign in to Prefect Cloud, and use the workspace switcher to open the `staging` workspace.
+1. Go to **Automations**, and then click **Add Automation**.
+
+### Specify the trigger
+
+An automation triggers whenever a specified condition is met.
+
+1. Select the **Flow run state** trigger type.
+1. Trigger whenever the `data-pipeline` flow has flow runs that enter the **Failed** state.
```
Comment (EmilRex) [inline docs/v3/tutorials/alerts.mdx:28]: We should alert on both `Failed` and `Crashed`.

File: docs/v3/tutorials/alerts.mdx (hunk 1 above)
Comment (EmilRex) [reply docs/v3/tutorials/alerts.mdx:2]: They can also be [defined with Terraform](https://registry.terraform.io/providers/PrefectHQ/prefect/latest/docs/resources/automation). We should mention that all those options are available.

**Review by EmilRex — COMMENTED** [review]
Another batch of high level comments.

File: docs/v3/tutorials/alerts.mdx (hunk 1 above)
Comment (daniel-prefect) [reply docs/v3/tutorials/alerts.mdx:2]: https://github.com/PrefectHQ/prefect/pull/16345/commits/9389e5c4f85fc5cfc4cdd8e926dccdc881cad9a6

Comment (daniel-prefect) [reply docs/v3/tutorials/alerts.mdx]: https://github.com/PrefectHQ/prefect/pull/16345/commits/ae06a7a2f8039f03f9cccd716609db113ee40831

**Review by EmilRex — APPROVED** [review]
Looks great!

File: docs/v3/tutorials/alerts.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+1. Name: `slack-alert`
+2. Description: `Send a Slack message when the 'data-pipeline' flow fails`
+3. Click **Save**.
+
+You've now created an automation that sends a Slack message whenever the `data-pipeline` flow in the `staging` workspace has a failed run.
+
+## Test the alert
+
+To test the alert, you need to trigger a flow run failure in the `staging` workspace.
+
```
[the remark this answers is not in the document]
Comment (daniel-prefect) [reply docs/v3/tutorials/alerts.mdx:57]: https://github.com/PrefectHQ/prefect/pull/16345/commits/7ccbff2fe3124fdfaf619f2ebe0c70210b805c3a

File: docs/v3/tutorials/alerts.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+title: Send alerts to Slack on failure
+description: Learn how to use automations to notify your team when a flow fails.
+---
+
+In the [Debug a data pipeline](/v3/tutorials/debug) tutorial, you learned how to troubleshoot flow runs that fail.
+In this tutorial, you'll learn how to set up a Slack alert which proactively notifies your team about failures.
```
[the remark this answers is not in the document]
Comment (daniel-prefect) [reply docs/v3/tutorials/alerts.mdx]: https://github.com/PrefectHQ/prefect/pull/16345/commits/618eb53594ea0a4aebf0fabfa4ecebe38832f20d

File: docs/v3/tutorials/alerts.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+1. Select the **Flow run state** trigger type.
+1. Trigger whenever the `data-pipeline` flow has flow runs that enter the **Failed** state.
+1. Click **Next**.
+
+Next, specify the action to take when the trigger condition is met.
+
+1. Choose the **Send a notification** action type
+1. In order to send a notification to Slack, you need to create a block.
+Blocks store configuration and provide an interface for external systems like Slack.
```
[the remark this answers is not in the document]
Comment (daniel-prefect) [reply docs/v3/tutorials/alerts.mdx]: https://github.com/PrefectHQ/prefect/pull/16345/files#diff-3f3a8e93e7921792a4841d85843decf49d38ffee6066944d0144b6cb69a425f3R37

File: docs/v3/tutorials/alerts.mdx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+```bash
+prefect cloud workspace set --workspace "<account handle>/staging"
+python simulate_failures.py --fail-at-run 1
+```
+
+After the script finishes, you should see ten Slack messages, one for each flow run failure.
+Each failure will contain a message similar to the following:
+
+> Prefect flow run notification<br/>
```
[the remark this answers is not in the document]
Comment (daniel-prefect) [reply docs/v3/tutorials/alerts.mdx]: https://github.com/PrefectHQ/prefect/pull/16345/commits/fe2fb669ab9b86b8dfd4d9b4882155ddaa27d143

File: docs/v3/tutorials/alerts.mdx [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+> Flow ID: 89053371-0d7e-4ef6-9a5f-63ea69689f66<br/>
+> Flow run ID: 33c2dac9-7d2a-48dd-a8cf-3894c6cd6063<br/>
+> Flow run URL: https:&#8203;//app.prefect.cloud/account/9b649228-0419-40e1-9e0d-44954b5c0ab6/workspace/a2127afa-ad10-4862-bdb0-5d065df64c8a/flow-runs/flow-run/33c2dac9-7d2a-48dd-a8cf-3894c6cd6063<br/>
+> State message: Flow run encountered an exception: Exception: Run failed
+
+You can click the URL to view the flow run in Prefect Cloud.
+
+## Next steps
+
+In this tutorial, you successfully used Prefect Cloud to set up a Slack alert.
```
[the remark this answers is not in the document]
Comment (daniel-prefect) [reply docs/v3/tutorials/alerts.mdx]: https://github.com/PrefectHQ/prefect/pull/16345/commits/c1519ae2c22522d7e65cd073ccc5b6c62639c27c

**Review by discdiver — APPROVED** [review]

**Pull request merged**